.claude/agents/CLAUDE.md

```
@@ -0,0 +1,245 @@
+### Sequência MCP Padrão
+1. **sequential-thinking** (análise)
+2. **archon** (task setup)  
+3. **serena** (codebase context)
+4. **Contextuais** (conforme necessidade)
+5. **desktop-commander** (implementação)
+
+---
+
+## 📚 Referências Rápidas
+
+### Documentação Principal
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)